tests/testthat/test-query-with-multibytes.R

```diff
@@ -0,0 +1,14 @@
+context("test-query-with-multibytes")
+
+test_that("Support multi-bytes queries", {
+  r <- plumber$new("files/query-with-multibytes.R")
+  res <- PlumberResponse$new()
+
+  req <- make_req("GET", "/msg", "\u53c2\u65701=\u4e2d\u6587&\u53c2\u65702=\u4f60\u597d")
```

Comment:
The query string cannot have raw UTF-8 characters in it. They must be URL-encoded. Also, this should have a leading `?`. So the query string should be:

```
"?%E5%8F%82%E6%95%B01=%E4%B8%AD%E6%96%87&%E5%8F%82%E6%95%B02=%E4%BD%A0%E5%A5%BD"
```

Same below.

Reply:
For reference:

> Octets must be encoded if they have no corresponding graphic character within the US-ASCII coded character set

From https://tools.ietf.org/html/rfc1738

Reply:
Thanks, I will make the change.